diff --git a/dist/index.js b/dist/index.js
index 0529104479f19d5131d5f3994e49e1f83fa66b1b..726436cbbb91ed02814c09ecb66544d099bd274a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -22,7 +22,7 @@ function studiocmsSocialPoster(opts) {
       user: [
         {
           title: {
-            "en-us": "Share to Social Media"
+            "en": "Share to Social Media"
           },
           description: "Share content on Social media",
           sidebar: "single",
